packages/main/src/plugin/extension-loader.ts

```diff
@@ -209,12 +209,12 @@ export class ExtensionLoader {
 
     let folders;
     // scan all extensions that we can find from the extensions folder
-    if (import.meta.env.PROD) {
-      // in production mode, use the extensions locally
-      folders = await this.readProductionFolders(path.join(__dirname, '../../../extensions'));
-    } else {
+    if (process.env.NODE_ENV && (process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test')) {
```

Comment:
I would prefer to not have these lines. I need to investigate to be able to use meta.env

because we've a lot of meta.env usage in Podman Desktop